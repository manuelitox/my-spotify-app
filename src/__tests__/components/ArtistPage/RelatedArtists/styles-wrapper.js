import React from 'react'
import { shallow } from 'enzyme'

import StyledWrapperRelatedArtists from 'components/ArtistPage/RelatedArtists/styles-wrapper'

describe('Component | StyledWrapperRelatedArtists', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledWrapperRelatedArtists />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})