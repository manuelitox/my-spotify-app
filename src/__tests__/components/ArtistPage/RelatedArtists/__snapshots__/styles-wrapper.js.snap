// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledWrapperRelatedArtists should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  display: grid;
  grid-template-columns: ",
          [Function],
          ";
  grid-gap: 30px;
  margin: 0 0 20px;
  @media only screen and (max-width: ",
          [Function],
          ") {
    grid-template-columns: 1fr 1fr;
  }
  @media only screen and (max-width: ",
          [Function],
          ") {
    grid-template-columns: 1fr;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "propTypes": Object {
        "inPopup": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
